README.md

```diff
@@ -138,3 +138,61 @@ sp_clean <- SCRI::clean_windows(sp_windows,
     ##             <Date>        <Date>             <Date>           <Date>          <Date>
     ## 1:      2021-02-12    2021-03-25         2021-04-25       2021-05-07      2021-05-07
     ## 2:      2021-03-12    2021-04-14               <NA>             <NA>      2021-04-14
+
+#### Step 3: Get records
+
+In this step, we will identify the events that occurred within each
+window after they have been **altered** or **cleaned**.
+
+**Select Dates Within Windows**: We use the `get_records()` function to
+identify all (or, if specified, only the first) records that fall within
+a specified window.
+
+This function identifies all records within the `records_table` that
+fall within a window specified in the `scri_trimmed` table in the start
+and end columns. It allows for the option to select only the first
+record within each window.
+
+**Parameters:**
+
+- `variable_name`: The name of the variable.
```

Comment:
Can you change the order in which these appear to match the order of the input function? So, scri_trimmed first? Also please take a look at the longer description I gave to that input in the function call and paste here